public/locations-list-map.js

```diff
@@ -264,19 +264,21 @@ function getFlatFilteredEntries(data, filters) {
     for (const cityName of Object.keys(cities).sort()) {
       const city = cities[cityName];
 
-      for (const entry of city.entries) {
+      city.entries.sort(function (a, b) {
+          return a.name.localeCompare( b.name );
+      }).forEach(function(entry) {
```

Comment:
Why switch from for..of to forEach?

Reply:
It seemed clearer to use chaining than to cram the entire comparator into the args for `for`, but I'm not really attached to any particular syntax. If you care a lot about this, I'm happy to change it.

Reply:
nah, I was just curious